spec/system/page_objects/pages/topic.rb

```diff
@@ -114,6 +115,32 @@ def fill_in_composer_title(title)
         @composer_component.fill_title(title)
       end
 
+      def fast_edit_button
+        find(".quote-button .quote-edit-label")
+      end
+
+      def click_fast_edit_button
+        find(".quote-button .quote-edit-label").click
+      end
+
+      def fast_edit_input
+        find("#fast-edit-input")
+      end
+
+      def select_text(selector, offset = 10)
+        js = <<-JS
```

Comment:
Disappointing that capybara doesn't have this built in :( This is definitely something that could be reused elsewhere. I would put this in `spec/support/system_helpers.rb` instead

Reply:
And maybe `select_text_range` is slightly better, supporting `start` as well as `offset` could be good too.